caster/lib/settings.py

```diff
@@ -71,7 +106,7 @@ def _find_natspeak():
         # EXECUTABLES
         "DEFAULT_BROWSER_PATH": "C:/Program Files (x86)/Mozilla Firefox/firefox.exe",
         "DOUGLAS_PATH": BASE_PATH + "/asynch/mouse/grids.py",
-        "ENGINE_PATH": _find_natspeak(),
+        "ENGINE_PATH": _validate_natspeak(),
```

Comment:
Validate should return the exe path , instead it returns  empty string as noted in the above comment.